<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect><x>0</x><y>0</y><width>600</width><height>400</height></rect>
  </property>
  <property name="windowTitle">
   <string>Descargador de Recursos HTML</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLineEdit" name="urlEdit">
     <property name="placeholderText">
      <string>Ingresar URL del sitio web</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="downloadButton">
     <property name="text">
      <string>Descargar recursos</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="logText">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
